rust/.cargo/config.toml

```diff
@@ -15,3 +15,6 @@ xclippy = [
 
 [build]
 rustflags = ["--cfg", "tokio_unstable"]
+
+[net]
+git-fetch-with-cli = true
```

Comment:
Is this safe to merge? @just-in-chang

Reply:
Not sure, don't know how CI does github authentication. Worried that it would break the docker build

Reply:
ok maybe we just open source the sdk repo 🤷‍♀️

Reply:
We can see if it fails first

Reply:
the build isn't even running lol